app/models/provider_interface/provider_user_form.rb

```diff
@@ -10,13 +10,17 @@ class ProviderUserForm
     validates :email_address, :first_name, :last_name, presence: true
     validates :email_address, email: true
     validates :provider_ids, presence: true
-    validate :email_is_unique
     validate :permitted_providers
 
     def build
       return unless valid?
 
-      @provider_user ||= ProviderUser.new
+      if existing_provider_user
+        @provider_user = existing_provider_user
+      else
+        @provider_user ||= ProviderUser.new
+      end
+
       @provider_user.first_name = first_name
       @provider_user.last_name = last_name
```

Comment:
Similar to my other comment, this overrides the first & last name for existing users. Technically this will allow a trolly school administrator to change the names for anyone in the system, even if they're not affiliated with the school.

Reply:
:+1: I think this should be left out of the _edit_ scenario so we only update providers and nothing else.

Reply:
@tijmenb I've distinguished between creating and updating here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/commit/92d097ad2b238071061b6ef4c708ee9381c84be2